lib/server/chat-llm.service.ts

```
@@ -37,7 +38,7 @@ export function createChatLLMService() {
  * @description Chat service that uses the LLM model to generate responses.
  */
 class ChatLLMService {
-  constructor() {}
+  constructor() { }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)